### ISSUE TITLE
html() does not behave as expected

### ISSUE THREAD

**Issue by xareelee** [cheeriojs/cheerio#1255]
> "version": "1.0.0-rc.2"
> "issue": "bug"

In the [README](https://github.com/cheeriojs/cheerio#attr-name-value-) it tells that:

```js
$('.apple').attr('id', 'favorite').html()
//=> <li class="apple" id="favorite">Apple</li>
```

but I run the following code:

```js
const cheerio = require('cheerio');
const content = `<ul id="fruits">
  <li class="apple">Apple</li>
  <li class="orange">Orange &amp; 你好</li>
  <li class="pear">Pear</li>
</ul>`;
const $ = cheerio.load(content);
const elem = $('.apple').attr('id', 'favorite').html(); 
console.log(elem); 
// expect => <li class="apple" id="favorite">Apple</li>
// but get => Apple
```

I think it is a bug, or the documentation is wrong.

**Comment by Gei0r** [comment]
The documentation is wrong. `.html()` only gives the inner html, so `Apple` is correct.
If you want to see the outer html, use `$.html(element)`, like this:

    const elem = $.html($('.apple').attr('id', 'favorite'));

https://runkit.com/embed/76sso9p4g2i3

**Comment by fb55** [comment]
Fixed by making the `1.0.0` branch the default one
